src/components/molecules/projectActivity/ProjectFundingContributorsItem.tsx

```diff
@@ -0,0 +1,56 @@
+import { Box, HStack } from '@chakra-ui/layout'
+import { HTMLChakraProps } from '@chakra-ui/system'
+import { Link } from 'react-router-dom'
+
+import { getPath } from '../../../constants'
+import { computeFunderBadges, getAvatarMetadata } from '../../../helpers'
+import { FunderWithUserFragment, Project, ProjectFragment } from '../../../types/generated/graphql'
+import { LinkableAvatar } from '../../ui'
+import { renderFunderBadges } from './renderFunderBadges'
+
+type Props = HTMLChakraProps<'div'> & {
+  project: ProjectFragment | Project
+  contributor: FunderWithUserFragment
+}
+
+export const ProjectFundingContributorsItem = ({ contributor, project, ...rest }: Props) => {
+  const avatarMetadata = getAvatarMetadata({ funder: contributor })
+  const funderBadges = computeFunderBadges({
+    creationDateStringOfFundedContent: project.createdAt,
+    funder: contributor,
+  })
+
+  return (
+    <Link
+      to={contributor?.user?.id ? getPath('userProfile', contributor.user.id) : '/'}
+      style={{ textDecoration: 'none', width: '100%' }}
+    >
+      <Box
+        px={6}
```

Comment:
tip (not a blocker) : For making any other component into a link, I prefer passing in " as ={Link} " prop, where `Link` is from `react-router-dom` .  One less dom node. 

```suggestion
      <Box
       as={Link}
       to={contributor?.user?.id ? getPath('userProfile', contributor.user.id) : '/'}
        px={6}
```